### PR TITLE
DOC-9637 - Seo Fixes

### PR DESCRIPTION
**Pull request by maria-robobug**

### CODE REVIEW

**Comment by maria-robobug** [comment]
Looks like these changed after 3.2.3

File: modules/hello-world/pages/cbc.adoc [hunk 1]
```diff
@@ -46,7 +46,7 @@ however, you can see all recent updates https://github.com/couchbase/libcouchbas
 * {lcb_api_link}/md_doc_cbc-pillowfight.html[cbc-pillowfight]:
 `cbc-pillowfight` is a stress test for Couchbase Client and Cluster.
 It creates a specified number of threads each looping and performing get and set operations within the cluster.
-* {lcb_api_link}/md_doc_cbc-n1qlback.html[cbc-n1qlback]:
+* {lcb_api_link}/md_doc_cbc_n1qlback.html[cbc-n1qlback]:
```
Comment (osfameron) [inline modules/hello-world/pages/cbc.adoc:49]: is this an seo fix? looks like just a link correction?
Oh, I suppose that will help SEO, fair enough, just checking I understood...

**Comment by osfameron** [comment]
> Looks like these changed after 3.2.3
is this something we could also automatically check on a new point release?
ideally the devs would tell us if URLs change, but I suspect they won't always know themselves.

(This is also a great use-case for a `api:` macro in antora...)

**Review by osfameron — APPROVED** [review]
Looks good, go go go!

(Questions inline mostly just me musing really, but all looks good 👍 )

**Pull request merged**